apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -369,7 +378,7 @@ class TopInstructionsCSP extends Component {
       ((this.props.viewAs === ViewType.Student && !studentHasFeedback) ||
         (this.props.viewAs === ViewType.Teacher &&
           !this.state.teacherViewingStudentWork));
-    const feedbackTabText = displayKeyConcept ? 'Key Concept' : msg.feedback();
+    const feedbackTabText = displayKeyConcept ? 'Key Concept' : i18n.feedback();
```

Comment:
Should 'Key Concept' be an 18n string too?

Reply:
Good catch - I'll make that update while we are here :)